ts/components/conversation/H5AudioPlayer.tsx

```diff
@@ -22,6 +27,24 @@ export const AudioPlayerWithEncryptedFile = (props: {
     }
   }, [playbackSpeed]);
 
+  useEffect(() => {
+    if (props.playableMessageIndex === props.nextMessageToPlay) {
+      player.current?.audio.current?.play();
+    }
+  });
```

Comment:
not sure the index logic will work well with incoming messages.

Reply:
I did clear the nextplay index here but maybe there should be more? 5a62fab